docker/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN useradd -u $USER_ID -g $USER_ID -s /bin/bash $USER_NAME
 ADD hub /opt/hub
 ADD run /opt/hub/bin/run
 RUN chown -Rv $USER_NAME:$USER_NAME /opt/hub
+RUN chown -Rv $USER_NAME:$USER_NAME /etc/hub
+RUN chown -Rv $USER_NAME:$USER_NAME /mnt/spoke
```

Comment:
Does this have the potential of taking a very long time in a spoke volume with lots of data?

Reply:
AFAIK this effects the permissions on files/directories inside the being-built-container-image not the full ones so I don't think it should take any longer to build

Reply:
This is broader than it needs to be, it should be fine tuned the files and sub-directories that hub uses only:
/etc/hub/hub.properties
/etc/hub/logback.xml
/mnt/spoke/read
/mnt/spoke/write

Reply:
I committed a change that should restrict permissions a bit more, as requested.

Reply:
for the sake of keeping this consistent with remote convo, further investigation it should be 


RUN chown -Rv $USER_NAME:$USER_NAME /etc/hub/hub.properties
RUN chown -Rv $USER_NAME:$USER_NAME /etc/hub/logback.xml
RUN chown -Rv $USER_NAME:$USER_NAME /mnt/spoke